src/pages/dataTableUtilities/getColumns.js

```diff
@@ -5,6 +5,8 @@
 
 import { tableStrings } from '../../localization';
 
+import COLUMN_TYPES from '../../widgets/DataTable/types';
```

Comment:
Need to do this for the `keys` also

Also I think `pages/datatableUtilities` is more suitable place

Reply:
Yeah, agreed. Done.